external_search.py

```diff
@@ -383,9 +383,9 @@ def _f(s):
 
         clauses = []
         if languages:
-            clauses.append(dict(terms=dict(language=languages)))
+            clauses.append(dict(terms=dict(language=list(languages))))
```

Comment:
Languages worked before, but exclude_languages didn't because the lane was created with a set. 

If it's a string it gets converted to a list here: https://github.com/NYPL-Simplified/server_core/blob/master/lane.py#L515
But if it's a set it stays a set.

Should I add a test for the string?